helm/charts/hpe-csi-driver/templates/nimble-csp.yaml

```diff
@@ -65,6 +65,11 @@ spec:
           resources:
             {{- toYaml .Values.csp.resources | nindent 12 }}
           imagePullPolicy: {{ .Values.imagePullPolicy | quote }}
+          env:
+            {{- if .Values.disableHostDeletion }}
+            - name: DISABLE_HOST_DELETION
+              value: {{ .Values.disableHostDeletion }}
```

Comment:
Environment variables are strings. Please follow the pattern for previous `disable*` parameters by setting `- value: "true"`.

```
json: cannot unmarshal bool into Go struct field EnvVar.spec.template.spec.containers.env.value of type string
```